packages/ui-extensions/docs/surfaces/admin/staticPages/admin-extensions.ext.doc.ts

```diff
@@ -97,8 +97,8 @@ const data: LandingTemplateSchema = {
     {
       type: 'Generic',
       title: 'Handling events',
-      sectionContent:
-        'Handling events in admin extensions are the same as you would handle them in a web app. You can use the `addEventListener` method to listen for events on the components or use the `on[event]` prop to listen for events from the components.',
+      sectionContent: `Handling events in UI extensions are the same as you would handle them in a web app. You can use the \`addEventListener\` method to listen for events on the components or use the \`on[event]\` property to listen for events from the components.
+        \n\nWhen using Preact, event handlers can be registered by passing props beginning with \`on\`, and the event handler name is case-insensitive. For example, the JSX \`<s-button onClick={fn}>\` registers \`fn\` as a "click" event listener on the button.`,
```

Comment:
this sounds to me like they're the same thing, so do we need to document them separately?

Reply:
@developit suggested the addition in [Slack](https://shopify.slack.com/archives/C051Q236X0U/p1748970608809929?thread_ts=1748969881.145309&cid=C051Q236X0U), so I created a [ticket](https://github.com/Shopify/admin-ui-foundations/issues/2928) for it, and then opened this PR.

Reply:
Yea. I can see how it feels a little redundant. But since Jason requested it, and I don't think it's harmful/incorrect, then I'm gonna ship it as-is, and we can iterate as needed.